modules/oe_content_tender/oe_content_tender.info.yml

```diff
@@ -0,0 +1,35 @@
+name: OpenEuropa Call for Tender Content
+description: Contains the EC corporate Call for tender content type.
+package: OpenEuropa
+type: module
+core: 8.x
+
+dependencies:
+  - composite_reference:composite_reference
+  - drupal:datetime
+  - drupal:link
+  - oe_content:oe_content
+  - oe_content:oe_content_departments_field
+  - oe_content:oe_content_reference_code_field
+  # todo: Remove oe_content_publication one project epic is merged!
```

Comment:
Todo has to be: `Set dependency oe_content:oe_content_documents_field instead of oe_content:oe_content_publication once project epic is merged!`

Reply:
Yep we will fix this in the next one, so we merge this one and unblock the track.